x-pack/plugins/security_solution/public/timelines/components/timeline/expandable_event/index.tsx

```diff
@@ -68,12 +98,20 @@ export const ExpandableEvent = React.memo<Props>(
 
     return (
       <>
-        <EuiText>{message}</EuiText>
+        {message && (
+          <EuiDescriptionList data-test-subj="event-message" compressed>
+            <EuiDescriptionListTitle>{i18n.MESSAGE}</EuiDescriptionListTitle>
+            <EuiDescriptionListDescription>
+              <LineClamp content={message} />
+            </EuiDescriptionListDescription>
+          </EuiDescriptionList>
+        )}
         <EuiSpacer size="m" />
```

Comment:
`<EuiSpacer>` should be moved inside `{message & (` otherwise, it creates big blank space between title and the details